.github/CONTRIBUTING.md

```diff
@@ -71,6 +70,18 @@ Explain local setup
 Explain CI setup
 ```
 
+In most cases the `<type>` should be one of the following:
+
+ * `feat` (new feature)
+ * `fix` (bug fix)
+ * `docs` (changes to documentation, README, …)
+ * `style` (css, missing semi colons, etc; no code change)
+ * `refactor` (restructuring docs, refactoring production code)
+ * `test` (adding missing tests, refactoring tests; no production code change)
+
+The `<scope>` will be the _class or component_ your changes are for. 
```

Comment:
The `<scope>` is one I can get behind, but to be really effective, a enumerated list would be needed, so you don't end up with `Docs:` vs `Documentation:` and `Env:` vs `Environment:`, or not knowing whether a change to `.env.example` goes under `Env:` or `Templates:`.